tests/unit/states/test_virt.py

```diff
@@ -225,163 +225,286 @@ def test_running(self):
                'changes': {},
                'result': True,
```

Comment:
any chance we can get some new test coverage for the new functions you added above?

Reply:
Those functions are used by the tested `virt.*running` already... isn't that enough?

Reply:
Just extracted unit tests for the `virt.*defined` out of the ones of the corresponding running tests.